backend/user_manager.js

```diff
@@ -79,6 +79,45 @@ module.exports = function (app, config, callback) {
         });
     });
 
+    app.post('/user/delete', checkLogin, checkAdmin, function (req, res) {
+        let userBeingDeleted = parseInt(req.body.userId);
+        // Only admin can delete the other user and nobody can delete admin
+        let userRequestedForDeletion = req.session.userId;
+        if (userBeingDeleted === userRequestedForDeletion) {
+            return res.status(406).send("Admin can't delete himself.");
+        }
+        const sql = `DELETE FROM users WHERE id=${userBeingDeleted}`;
+        const db = mysql.createConnection(config.database);
+        db.query(sql, function (err, results) {
+            db.end();
+            if (!err) {
+                res.status(200).send('User has been successfully deleted.')
+            } else {
+                Object.entries(err);
+                res.status(500).send('Failed to delete the requested user because of db related error.')
+            }
+        });
+    });
+
+    app.post('/user/toggle/activation', checkLogin, checkAdmin, function (req, res) {
+        let userActivationBeingToggled = parseInt(req.body.userId);
+        let userRequestedForChangeInActivation = req.session.userId;
```

Comment:
Can we rename this variable to `currentUser` or `loggedInUser` ?

Reply:
yeah, sure.